src/services/ProductExports.ts

```diff
@@ -119,9 +124,6 @@ export async function updateProdExportInDb(apiReq: api.ProdExportUpdateReq) {
     if (apiReq.transportId) {
       await db.updateProdExportField(apiReq.uuid, 'transportid', apiReq.transportId);
     }
-    if (apiReq.datetime) {
```

Comment:
i thought we were letting them change datetime? and then should we let them change recorder uuid?